api/src/main/java/com/viaversion/viaversion/api/minecraft/entities/EntityTypes1_10.java

```diff
@@ -234,13 +264,12 @@ public EntityType getType() {
             return type;
         }
 
-        public static Optional<ObjectType> findById(int id) {
-            if (id == -1)
-                return Optional.empty();
+        public static Optional<ObjectType> findById(final int id) {
+            if (id == -1) return Optional.empty();
             return Optional.ofNullable(TYPES.get(id));
         }
 
-        public static Optional<EntityType> getPCEntity(int id) {
+        public static Optional<EntityType> getEntityType(final int id) {
```

Comment:
The optional returning methods could just be made nullable instead

Reply:
Yup thanks, fixed it.